### PR TITLE
Add sitemap

### PR DESCRIPTION
**Pull request by giankotarola**

### CODE REVIEW

File: static/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+</url>
+<url>
+  <loc>https://blog.fleek.co/page/1/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/wordpress+fleek/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.64</priority>
```
Comment (Blockchainbrett) [inline static/sitemap.xml]: can we make this higher priority?

File: static/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/fleek-nextJS/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.64</priority>
```
Comment (Blockchainbrett) [inline static/sitemap.xml]: can we make this higher priority?

File: static/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+</url>
+<url>
+  <loc>https://blog.fleek.co/category/guide/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/Gatsby-Fleek/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.64</priority>
```
Comment (Blockchainbrett) [inline static/sitemap.xml]: can we make this higher priority?

File: static/sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/wordpress/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.51</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/fleek-create-react-app/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.51</priority>
```
Comment (Blockchainbrett) [inline static/sitemap.xml]: can we make this higher priority?

File: static/sitemap.xml [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/fleek+helloWorld/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.51</priority>
```
Comment (Blockchainbrett) [inline static/sitemap.xml]: can we make this maybe the highest priority?

File: static/sitemap.xml [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+</url>
+<url>
+  <loc>https://blog.fleek.co/category/getting-started/</loc>
+  <lastmod>2020-04-08T18:27:16+00:00</lastmod>
+  <priority>0.41</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/getting-started/</loc>
+  <lastmod>2020-04-08T18:27:16+00:00</lastmod>
+  <priority>0.41</priority>
```
Comment (Blockchainbrett) [inline static/sitemap.xml]: can we make this higher priority?

File: static/sitemap.xml (hunk 1 above)
Comment (giankotarola) [reply static/sitemap.xml]: change in 0099621

File: static/sitemap.xml (hunk 2 above)
Comment (giankotarola) [reply static/sitemap.xml]: change in 0099621

File: static/sitemap.xml (hunk 3 above)
Comment (giankotarola) [reply static/sitemap.xml]: change in 0099621

File: static/sitemap.xml (hunk 4 above)
Comment (giankotarola) [reply static/sitemap.xml]: change in 0099621

File: static/sitemap.xml (hunk 5 above)
Comment (giankotarola) [reply static/sitemap.xml]: change in 0099621

File: static/sitemap.xml (hunk 6 above)
Comment (giankotarola) [reply static/sitemap.xml]: change in 0099621

**Pull request merged**